src/index.js

```diff
@@ -40,12 +40,24 @@ const normalizeId = (id) => {
   return id;
 };
 
+const getEntries = ({ entries }) => {
+  if (!entries) {
+    return [];
+  }
+
+  if (Array.isArray(entries)) {
+    return entries;
+  }
+
+  return Object.keys(entries).map(key => ({ find: key, replacement: entries[key] }));
+};
+
 export default function alias(options = {}) {
   const resolve = Array.isArray(options.resolve) ? options.resolve : ['.js'];
-  const entries = options.entries?options.entries:[];
+  const entries = getEntries(options);
 
   // No aliases?
-  if (!entries || entries.length === 0) {
+  if (!entries.length === 0) {
```

Comment:
You've accidently negated the entires length === 0 here, no? As such it should never work if its configurated correctly and the opposite way around :smile:

Reply:
good catch, my intention was to remove the first part of this OR, but I've accidentally missed this `!`